src/pudl/package_data/notebooks/how_to_access_pudl.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<p>The <code>sqlite3</code> package is also part of the Python standard libraries, so if someone is familiar with it already, it will be available anywhere Python is installed, which might be useful to note.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>we talked about showing the tools that we actually use. i don't think any of us use the <code>sqlite3</code>  package regularly. and I noted above that there is a ton of ways to access sqlite db's. i don't think we can or should show folks every way.</p>